src/daft-core/src/array/fixed_size_list_array.rs

```diff
@@ -132,6 +132,30 @@ impl FixedSizeListArray {
         ))
     }
 
+    pub fn iter(&self) -> Box<dyn Iterator<Item = Option<Series>> + '_> {
```

Comment:
So typically for `iter` you would implement the `IntoIterator` [trait](https://doc.rust-lang.org/std/iter/trait.IntoIterator.html) on the type. This function then returns a struct that would implement `Iterator` https://doc.rust-lang.org/std/iter/index.html#implementing-iterator.

One you do this, you can use the Array in for loops and what not.